### DISCUSSION TITLE
How to migrate users from another SFTP tool GoAnywhere to SFTPGo

### DISCUSSION THREAD

**Discussion by rashmimaharana** [drakkan/sftpgo#523] [Q&A]
Hi @drakkan and Team,

We have a requirement to migrate users from GoAnywhere to SFTPGo. How can we import/onboard users including their metadata, password and others from GoAnywhere to SFTPGo. Any workaround for this.

Thanks,
Rashmi Maharana

**Comment by drakkan** [comment]
Hi,

I've never used GoAnywhere, you have to figure it out yourself, sorry. Maybe you can export the users in some format and then import them in SFTPGo using the REST API, you likely need to write a specific import script

**Reply by rashmimaharana** [reply]
Thanks @drakkan 
We have import functionality in SFTPGo. If we export the data from GoAywherein exact format what SFTPGo needs, then we can import the users. Right ?

**Reply by drakkan** [reply]
yes, if you generate a json backup file in the SFTPGo format it will work. The SFTPGo data structures are documented here:

- [development version](https://github.com/drakkan/sftpgo/blob/main/httpd/schema/openapi.yaml)
- [2.1.x branch](https://github.com/drakkan/sftpgo/blob/2.1.x/httpd/schema/openapi.yaml)
